tests/phpunit/data/themedir1/block-theme-with-hooked-blocks/blocks/hooked-after/block.json

```diff
@@ -1,4 +1,6 @@
 {
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"title": "Hooked Block After",
```

Comment:
What's the nature of the change adding the `title`? Can you leave a note in the description sharing context for why they are added here and why adding them is safe?

Reply:
Done, I added a note about these changes to the description.